test/Engine/ProtoTest/Associative/MethodResolution.cs

```diff
@@ -388,6 +388,30 @@ public void MAGN5729_Repro_Simple()
             thisTest.Verify("r3", new object[] { true, false });
         }
 
+        [Test]

```

Comment:
What is the behaviour if a variable inside the function body is used that has the same name as the function? Is that also fixed here?
```
def foo(a)
{
	foo = 2;
	return = foo+1;
};
```